File: Python/Program/Human.py
class Human:
    __nik = ""
    __nama = ""
    __gender = ''

    def __init__(self, nik = "", nama = "", gender = ''):
        self.__nik = nik
        self.__nama = nama
        self.__gender = gender
    
    # Get-set NIK
    def setNIK(self, nik):
        self.__nik = nik

    def getNIK(self):
        return self.__nik

    # Get-set Nama
    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    # Get-set Gender
    def setGender(self, gender):
        self.__gender = gender

    def getGender(self):
        return self.__gender

File: PHP/Program/Human.php
<?php

class Human
{
    private $nama = "";
    private $nik = "";
    private $gender = '';

    public function __construct($nama = "", $nik = "", $gender = ''){
        $this->nama = $nama;
        $this->nik = $nik;
        $this->gender = $gender;
    }

    // Get-set Nama
    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }

    // Get-set NIK
    public function setNIK($nik){
        $this->nik = $nik;
    }

    public function getNIK(){
        return $this->nik;
    }

    // Get-set Gender
    public function setGender($gender){
        $this->gender = $gender;
    }

    public function getGender(){
        return $this->gender;
    }
}

?>